ExpectedSlotConsoleMessages.json

```diff
@@ -1,7 +1,69 @@
-{
-    "SlotMessages": [
-      "gameLoadComplete",
-      "message type: \"balanceUpdate\"",
-      "message type: \"stakeUpdate\""
+  {
+    "testmessages": [
+      {
+        "supportedcurrencies": [
+          "currency:gbp",
+          "currency:eur"
+        ]
+      },
+      {
+        "gameLoadMessages": [
+          "gameLoadComplete"
+        ]
+      },
+      {
+        "balanceUpdateMessage": [
+          "balanceUpdate"
+        ]
+      },
+      {
+        "stakeUpdateMessage": [
+          "stakeUpdate"
+        ]
+      }, 
+      {
+        "depositFlowMessage": [
+          "balanceUpdate"
+        ]
+      }, 
+      {
+        "gameHelpMessage": [
+          "menuOpened", 
+          "menuClosed", 
+          "gameHelp"
+        ]
+      }, 
+      {
+        "payTableMessage": [
+          "toggle paytable"
+      ]
+    },
+    {
+      "soundCheckMessageToolBar": [
+        "sound:true",
+        "sound:false"
+      ]
+    }, 
+    {
+      "soundCheckMessageGame": [
+        "settingName:sound settingValue:on",
```

Comment:
might need changing to match what the full error is - this needs to be an exact match

Reply:
So in this case you want to match
"settingName":"sound","settingValue":"on"